Addition of release documentation (#1396)
vitessio/website: content/en/docs/releases/release-notes.md

---
title: Release Notes
description: Learn how release notes are generated
weight: 3
---

The [release notes](https://github.com/vitessio/vitess/releases) of Vitess describe what a new release is about. We list the new features, bug fixes, important announcement, deprecation notice, etc.

> Since we have a release note document for each release, it was important to us that we create a tool that automatically generate them for us.

Reviewer: ```suggestion
Since we publish release notes for each release, it was important to us to have a tool that can automatically generate them for us.
```

> The tool can be found in [this package](https://github.com/vitessio/vitess/tree/main/go/tools/release-notes).

Reviewer: ```suggestion
This tool can be found in [this package](https://github.com/vitessio/vitess/tree/main/go/tools/release-notes).
```

> ### How to use release note generation tool?

Reviewer: ```suggestion
### How to use the release notes generation tool
```
